src/App.tsx

```diff
@@ -10,33 +11,27 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tabId: string) => {
+    const selected = tabs.find((tab) => tab.id === tabId);
```

Comment:
```suggestion
    const selected = tabs.find((tab) => tab.id === tabId) || tabs[0];
```
![image](https://user-images.githubusercontent.com/96409563/205143258-0da8eb5c-e852-4317-9ff5-b090c0cd9431.png)